cypress/support/on-rails.js

```diff
@@ -0,0 +1,74 @@
+// CypressOnRails: dont remove these command
+Cypress.Commands.add('appCommands', function (body) {
+  cy.log('APP: ' + JSON.stringify(body))
```

Comment:
It's ok to have this one?

Reply:
mmm, I'm not sure, this comes with CypressOnRails. I don't know if it's being used by it internally tbh.